pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -53,7 +55,7 @@
 
 # dependencies requires for stable backports
 
-, rust-cbindgen_0_14_1
```

Comment:
Last nitpick: SInce we are touching these lines how about we merge this block with the one above at line 19-22? Would be more consistent and we wouldn't have two "blocks" for backports :)